inventory/sample-ha.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,56 @@
+# 'hosting_infrastructure' is used to drive the correct behavior based
+# on the hosting infrastructure, cloud provider, etc. Valid values are:
+# - 'openstack'
+# - 'aws' (Coming Soon)
+# - 'azure' (Coming Soon)
+hosting_infrastructure: aws
+
+# AWS Provisioning Variables
+# use the "-e" option to set "aws_ssh_public_key"
+aws_image_name: <ec2 AMI>
+aws_num_masters: 3
+aws_num_nodes: 5
+aws_num_infra: 3
+aws_access_key: <Access key ID>
+aws_secret_key: <Secret access key>
+aws_region: <AWS Region>
```

Comment:
Can we fill in some sample values here - i.e.: don't have to be valid, but something that looks like the actual values

Reply:
Sure, will do